### DISCUSSION TITLE
Struggle to get set up - Can you spot my error?

### DISCUSSION THREAD

**Discussion by rmfm** [robmarkcole/HASS-plate-recognizer#90] [Q&A]
Hi all, feel quite silly here as I'm following a video (Everything smart home) and the read me (Github), however I'm getting a invalid config warning. :-(

I've installed the integration via HACS, restarted a few times.
I never seem to generate the service entity to be able to trigger it.

So if someone could tell me what I the donut have done wrong that would be super.

<img width="743" alt="image" src="https://github.com/robmarkcole/HASS-plate-recognizer/assets/85053638/b0e7a2c8-7421-4f16-9daf-0b5df1fe43c2">
FYI on the everything smart home I've noted the correction on always_save_last_jpg to always_save_latest_file
<img width="758" alt="image" src="https://github.com/robmarkcole/HASS-plate-recognizer/assets/85053638/7b0d65ca-1115-492d-b1d9-bf886a8809f3">

**Comment by robmarkcole** [comment]
That video is old right? Best to ask on forums

**Reply by rmfm** [reply]
It is a bit old yes, but I think it's something I'm missing. Like I'm doing something basic wrong IMO.
I'll try to the forum on EverythingSmartHome assuming it has one :)

I take it there's nothing jumping out in the config I have shown that looks wrong to you? I've mostly taken the one from the read me.

**Comment by rmfm** [comment]
This is why I'm thinking it's me in the config rather than any real issue.

<img width="358" alt="image" src="https://github.com/robmarkcole/HASS-plate-recognizer/assets/85053638/95db5895-0d17-49d3-98c7-d2eb7e2066cc">

**Reply by robmarkcole** [reply]
Comment out parts of the config until you clear the error
